test/html.js

```diff
@@ -415,26 +415,32 @@ describe('HTML Parser', function () {
 				root.firstChild.getAttribute('alt').should.eql('«Sogno');
 				root.firstChild.rawAttributes.alt.should.eql('&laquo;Sogno');
 			});
-			it('shoud not decode text', function () {
+
+			it('should not decode text from parseHTML()', function () {
 				// https://github.com/taoqf/node-html-parser/issues/33
-				const root = parseHTML(`<html>
-<body>
-<div id='source'>
-&lt;p&gt;
-This content should be enclosed within an escaped p tag&lt;br /&gt;
-&lt;/p&gt;
-</div>
-</body>
-</html>`)
-				root.toString().should.eql(`<html>
-<body>
-<div id='source'>
-&lt;p&gt;
-This content should be enclosed within an escaped p tag&lt;br /&gt;
-&lt;/p&gt;
-</div>
-</body>
-</html>`);
+				const content = `&lt;p&gt; Not a p tag &lt;br /&gt; at all`;
+				const root = parseHTML(`<div>${content}</div>`);
+				root.childNodes.should.have.length(1);
+
+				const divNode = root.firstChild;
+				divNode.childNodes.should.have.length(1);
+
+				const textNode = divNode.firstChild;
+				textNode.rawText.should.eql(content);
+			});
```

Comment:
This test ultimately did not need to be rewritten, however, I left my change in because it is more complete and easier to read. I also renamed it so that it is easier to understand what's happening. If you'd prefer, it can be changed back.